docs/guides/utilizing-forms.md

```diff
@@ -0,0 +1,71 @@
+---
+id: utilizing-forms
+title: Building forms with Modeler
+---
+
+:::note
+The initial release of Camunda Forms includes a debut minimal feature set, which will be expanded with upcoming versions.
+
+If using with Camunda Platform, note that the Camunda Forms feature was added with the 4.7.0 release of the Camunda Modeler. Therefore, they can be used within BPMN diagrams running on Camunda Platform version 7.15.0 or later.
+:::
+
+# Overview
+
+The Camunda Forms feature allows you to easily design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
+
+# Quickstart
+
+## Create new form
+
+To start building a form, log in to your [Camunda Cloud](./getting-started/create-camunda-cloud-account.md) account or open [Camunda Modeler](./components/modeler/what-is-camunda-modeler.md) and take the following steps:
+
+1. Click on the **Forms** tab at the top of the page or open the **File** menu in Modeler.
+2. Click the blue **Create New Form** button.
+
+## Build your form
+
+Now you can start to build your Camunda form. First, let's name our form by clicking the three vertical dots next to the text reading **New Form** in the top left corner of the page. Select **Rename** to give your form a new name. In this example, we'll analyze a form to help with a task in obtaining an email message.
+
+![form email example](./img/form-email-example.png)
+
+Add your desired elements from the palette on the left side by dragging and dropping them onto the canvas.
+
+![form palette](./img/form-palette.png)
+
+Within Forms, we have the option to add text fields, numerical values, checkboxes, radio elements, selection menus, text components, and buttons.
+
+:::note
+Within Camunda Platform, you can also analyze [embedded forms](https://docs.camunda.org/manual/latest/reference/forms/embedded-forms/).
+:::
+
+In the properties panel on the right side of the page, view and edit attributes that apply to the selected form element. For example, apply a minimum or maximum length to a text field, or require a minimum or maximum value within a number element. In this case, we have labeled the field, described the field, and required an input for our email message.
+
+![email properties](./img/form-properties-email.png)
+
+Refer to the [Camunda Forms reference material](https://docs.camunda.org/manual/latest/reference/forms/camunda-forms/) to explore all form elements and configuration options in detail.
+
+## Save your form
+
+To save your form in Camunda Cloud, click the blue **Save** button in the top right corner of the page.
+
+To save your form in Camunda Platform, click **File > Save File As...** in the top-level menu. Select a location on your file system to store the form as `.form` file. You can load that file again by clicking **File > Open File...**.
+
+## Connect your form to a BPMN diagram
+
+Click on the **Forms** tab again to view a list of all the forms you've created.
```

Comment:
Differs entirely in desktop modeler, as it does not help with selecting a form.

Reply:
Pulled the process for Platform out in a note admonition in the latest commit -- is there additional crucial information for Platform you would add here?